### DISCUSSION TITLE
about/index.html

### DISCUSSION THREAD

**Discussion by giscus** [SouthFox-D/SouthFox-D.github.io#91] [博客评论]
# about/index.html

欢迎点击这个页面，不管是自动执行的程序还是有血有肉的生物。在互联网闲逛到一些个人博客时，我通常都会首先点开个人博客（尤其是首页都是些高深技术文章的那种博客）以便来了解博主，但想想这样做有点不太「公平」，毕竟我的博客还没有一个关于页面。

所以现在（2023-04-28）我就写了。

但虽说是关于页面

https://blog.southfox.me/about/

**Comment by novob** [comment]
好帅的博客呀

**Comment by ScDuckXu** [comment]
真的，这Theme太帅了。太帅了，这使我的电脑回到2005年。

**Reply by virelyx258** [reply]
居然在这里捕捉到了 DuckXu 😜

**Comment by 0xis-cn** [comment]
刚发现原来右边能换字体TuT！小安那里我也换掉了，像素字体对我有些难读。

**Comment by Rayme** [comment]
你的网站风格太帅了！
